tests/playwright/Report.md

```diff
@@ -0,0 +1,308 @@
+
+- [Cypress Evaluation](https://www.mediawiki.org/wiki/User:SimTran/Cypress_vs_WebdriverIO:_What_are_the_differences%3F) by Sim T.H. Tran
+- [Puppeteer Evaluation](https://www.sohamp.dev/blog/2020-08-24-wmf-gsoc2020/) by Soham Parekh
+
+So, this brings us to the current framework being evaluated against WebdriverIO, which is Playwright. The evaluations progress involved the following steps;
+
+- Configuring and setting up of Wikimedia Core to run on continuous integration, in this case on [Github Actions](https://docs.github.com/en/actions).
+- Implementation of all MediaWiki Core tests in Playwright.
+- Running MediaWiki Core tests in WebdriverIO.
+- Analysis of both WebdriverIO and Playwright tests in terms of speed and stability.
+
+## WebdriverIO
+
+WebdriverIO is a test automation framework that allows you to run tests based on the WebDriver protocol and Appium automation technology. WebdriverIO is known to be;
+
+- Extendable: Adding helper functions, or more complicated sets and combinations of existing commands is simple and really useful.
+- Compatible: WebdriverIO can be run on the WebDriver Protocol for true cross-browser testing as well as Chrome DevTools Protocol for Chromium-based automation using Puppeteer.
+- Feature Rich: The huge variety of community plugins allows you to easily integrate and extend your setup to fulfill your requirements.
+
+The above points are just a few of WebdriverIO capability. To explore more WebdriverIO capabilities, you can check out the WebdriverIO and [MediaWiki documentation](https://www.mediawiki.org/wiki/Selenium).
+
+WebdriverIO provides the ability and options to run commands in both asynchronous and synchronous operations. For asynchronous operations, you can use the JavaScript usual async/await, however for synchronous, it can be done through [node-fibers](https://www.npmjs.com/package/fibers). Both of these options have their own benefits and issues which can be found on the [WebdriverIO](https://webdriver.io/docs/sync-vs-async) website.
+
+### Sample WebdriverIO Code
+
+The code snippet below navigates to wikipedia.org in Chrome and checks if the correct title is being displayed.
+
+```js
+const assert = require('assert');
+const { remote } = require('webdriverio');
+
+(async () => {
+ const browser = await remote({
+        capabilities: {
+            browserName: 'chrome'
+        }
+  });
+  await browser.url('https://www.wikipedia.org/');
+
+  const title = await browser.getTitle();
+  assert(title === 'The Free Encyclopedia');
```

Comment:
Is there a reason to use `assert` here and `expect` in the next code sample? I think picking one and using it in both places would be better.